toolkit/docs/how_it_works/3_package_building.md

```diff
@@ -124,6 +126,8 @@ Edges in the graph represent dependencies. A package cannot be installed (`run`)
 
 Once all packages have been added to the graph, the inter-package dependencies are added. For each `BuildRequires` in a package an edge is created from the current `build` node to the `run` node associated with the required package. The same is done for each `Requires`, but from the current `run` node instead of the `build` node.
 
+Edges will not be created for `Requires` between two `TypeRun` nodes that have the same `RpmPath` since the dependency will automatically be met when the rpm is installed. If the edges were created it could introduces extra cycles that would need to be solved.
```

Comment:
>TypeRun` nodes [](start = 54, length = 14)

Is the `TypeRun` node the same as the `run` node mentioned above? If yes, let's use just one way of calling these to avoid confusion. If not, could you add a sentence first explaining what the `TypeRun` node is?